src/test/scala/edu/berkeley/ce/rockslicing/BlockVTKSpec.scala

```diff
@@ -85,6 +162,20 @@ class BlockVTKSpec extends FunSuite {
     assert(faceConnectivities == vtkBlock.connectivityMap)
   }
 
+  test("Connectivities for each face should be in terms of vertex ID's (seven-sided block)") {
+    val vtkBlock = BlockVTK(sevenSidedBlock)
+    val faceConnectivities = Map(
+      face1_s7 -> List[Int](7, 8, 5, 4),
```

Comment:
If we're going for consistency with our other code, we could use `Seq` here instead.